eos-updater-flatpak-installer/docs/eos-updater-flatpak-installer.8

```diff
@@ -10,6 +10,8 @@ eos\-updater\-flatpak\-installer — Endless OS Updater Flatpak Installer
 .IX Header "SYNOPSIS"
 .\"
 \fBeos\-updater\-flatpak\-installer [\-m \fPmode\fB] [\-p]
+.PP
+\fBeos\-updater\-flatpak\-installer \--dry-run
```

Comment:
You need to escape each hyphen, so this should end with `\-\-dry\-run`. Isn’t troff/groff fun?